pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -2613,6 +2613,15 @@ type StorageScopeSpec struct {
 	StorageClassDeviceSets []StorageClassDeviceSet `json:"storageClassDeviceSets,omitempty"`
 	// +optional
 	Store OSDStore `json:"store,omitempty"`
+	// +optional
+	// RestartInterval defines the time for which the OSD pods, that failed with zero exit code, will sleep before restarting.
+	// This is useful in scenarios like OSD flapping, when OSDs are marked down more than 5 times in 600 seconds by Ceph.
```

Comment:
Flapping is the only scenario this is for, right?
```suggestion
	// This is needed for OSD flapping, when OSDs are marked down more than 5 times in 600 seconds by Ceph.
```